better-words/words/aave.yml

```diff
@@ -0,0 +1,31 @@
+# A default response, that prefixes all messages from the bot within this category
+# TODO: actually support opting out
+# TODO: Maybe have a specific standard opt out response section
+response: Please be aware that this word or phrase originated as part of African American Vernacular English (AAVE), colloquially referred to as ebonics. In general, we recommend avoiding AAVE to prevent appropriation of black culture. However, as most languages do, many words from this vernacular have spread into common english communication. If you are African American, or for some other reason feel comfortable using AAVE, you may opt out of this category at any time by messaging me _"opt out aave"_
```

Comment:
also just something to think about - we probably want to have opting out of categories and words both! and make it clear to people they can do either (so like, adding something to the message we send to ppl explaining how to do both)